public/app/features/admin/ldap/LdapSettingsPage.tsx

```diff
@@ -308,21 +310,24 @@ export const LdapSettingsPage = () => {
                     id="search_filter"
                     placeholder={t('ldap-settings-page.search_filter.placeholder', 'example: cn=%s')}
                     type="text"
-                    {...register('settings.config.servers.0.search_filter', { required: true })}
+                    {...register(`${serverConfig}.search_filter`, { required: true })}
                   />
                 </Field>
                 <Field
                   label={t('ldap-settings-page.search-base-dns.label', 'Search base DNS *')}
                   description={t(
                     'ldap-settings-page.search-base-dns.description',
-                    'An array of base dns to search through; separate by commas or spaces.'
+                    'An array of base dns to search through; separate by spaces.'
                   )}
                 >
                   <Input
                     id="search-base-dns"
                     placeholder={t('ldap-settings-page.search-base-dns.placeholder', 'example: "dc=grafana.dc=org"')}
                     type="text"
-                    {...register('settings.config.servers.0.search_base_dns', { required: true })}
+                    value={watch(`${serverConfig}.search_base_dns`, []).join(' ')}
+                    onChange={({ currentTarget: { value } }) =>
+                      setValue(`${serverConfig}.search_base_dns`, value.split(' '))
+                    }
                   />
```

Comment:
Do you think we can find a better UI component for managing a list of strings? Maybe we can reuse the one for managing the roles? I think it's a big risk to go with the current approach that splits by space char because some existing LDAP configs might already have spaces in some search-base-dns items. And in that case, saving the config to the database will break the LDAP config.

Reply:
I'll take another look. I think we can use [MultiSelect](https://developers.grafana.com/ui/canary/index.html?path=/story/forms-select--multi-select-basic-with-select-all)

Reply:
I've replaced it with a MultiSelect and now it behaves similarly to the root ca certificates multiselect.

Additionally I addressed your comment left on a previous pr - [link](https://github.com/grafana/grafana/pull/93173#pullrequestreview-2292818906)